modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -996,6 +996,45 @@ CV__DNN_EXPERIMENTAL_NS_BEGIN
                              CV_OUT std::vector<int>& indices,
                              const float eta = 1.f, const int top_k = 0);
 
+     class CV_EXPORTS_W Model : public Net
```

Comment:
Please document new class and it's methods (see other classes in this header for doxygen syntax reference).

Reply:
done